### DISCUSSION TITLE
Dolce Far Niente - Não fazer nada vai te ajudar um dia

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#109] [Talks in Portuguese]
**Palestrante:** Lucas Fieri

**Assunto**:

Vamos falar um pouco sobre como “o prazer de não fazer nada” pode ajudar a desenvolver melhor seu trabalho. Vamos também abordar alguns tópicos como ócio criativo e como eles estão diretamente ligados a sua produtividade e desenvolvimento profissional.

**Comment by lucasfieri** [comment]
### PT
Oioi galera!! Sou Lucas mais conhecido por Lukita e vamos conversar quinta-feira um pouco sobre ócio criativo, jornada de trabalho e como isso influencia sua produtividade :)
Fiquem a vontade para mandar dúvidas e indicações por aqui ou pelas minhas redes sociais. 

### ES
¡¡Hola chicos!! Soy Lucas más conocido como Lukita y el jueves vamos a hablar un poco sobre el ocio creativo, el horario laboral y cómo influye en tu productividad :)
Siéntase libre de enviar preguntas e indicaciones aquí o en mis redes sociales.

https://www.linkedin.com/in/lucasfieri/
https://twitter.com/eu_lukita (not a tech account)

**Comment by lucasfieri** [comment]
Oioi, tem aqui o link com os slides pra quem quiser rever :)

https://docs.google.com/presentation/d/1Cp35nSSgyPdStp2htC5f9ENL1E7SWRPmQvgIJm8du5o/edit?usp=sharing

**Comment by patriciaverso** [comment]
O tema da palestra é importante. No entanto, eu senti um certo incômodo com a motivação do "não fazer nada".
Em grande parte da palestra foi dito que não fazer nada te daria mais foco para fazer mais depois, ou como fazer menos te faz mais produtivo. No mundo capitalista que infelizmente habitamos eu claro que produtividade em importante, mas não deve ser o objetivo do ócio. O ócio é pra descansar, viver a vida, olhar pro céu, não pra poder no futuro dar mais dinheiro pra empresa que te contrata.
Como disse, o tema é importante, mas sinto que a palestra caiu em como ser produtivo.

Além disso, um comentário: as pessoas antigamente não almoçavam em casa porque "tinham mais tempo". Elas almoçavam em casa porque só quem trabalhava fora era o homem e a mulher ficava em casa disponível pra fazer o almoço.
